### DISCUSSION TITLE
The direct linking to a job is not working | how can people share the job?

### DISCUSSION THREAD

**Discussion by johanvanwambeke** [bullhorn/career-portal#512] [General]
I used the career portal, published it on netlify and it does not work. It works in localhost.
What is going on here?

https://jobs.kwery.be -> works
https://jobs.kwery.be/jobs/113 -> does not work

**Comment by johanvanwambeke** [comment]
It had to do with netlify hosting, add a _redirect file with `/* /index.html  200`

**Comment by pbearne** [comment]
I love the styling you have done on this site 
one of the best OSCP styles I have seen

one issue I do see is that Google does not see the details of the jobs so you are not getting listed by Google 
https://search.google.com/test/rich-results/result?id=-grM0JYUvpLh2a4uI7l-Vw

Here is the result from Matador Jobs Client https://search.google.com/test/rich-results/result?id=6ea8fRG9NqCvgLV1OvGtGw

You need to get the advance hosting working to fix this

**Reply by johanvanwambeke** [reply]
Hey Bearne, your solution is great, it's so painfull I only found it to late... would have saved us so much trouble. Notice I have the picture and personal information of the reps on the site... picking them from the landingpage 😮‍💨 .... realy you could have saved me some grey hairs.

**Reply by pbearne** [reply]
We try to spread the word the best we can :-)
The recruiter profiles are WP cpt's that are linked via the recruiter emails 

Happy to do a demo as we have lots of upgrades that may make it worth redoing the job board.
